internal/pkg/service/buffer/store/model/column/column_test.go

```diff
@@ -110,3 +110,68 @@ func TestColumn_Header(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, string(headerMarshalled), val)
 }
+
+func TestColumn_Template_Body(t *testing.T) {
+	t.Parallel()
+
+	c := column.Template{Language: column.TemplateLanguageJsonnet, Content: "Body('key1.key2')"}
+
+	val1 := orderedmap.New()
+	val1.Set("key2", "val2")
+	body := orderedmap.New()
+	body.Set("key1", val1)
+	body.Set("key3", "val3")
+
+	header := http.Header{}
+	header.Set("Content-Type", "application/json")
+	header.Set("Content-Encoding", "gzip")
+
+	val, err := c.CsvValue(column.ImportCtx{Body: body, Header: header})
+	assert.NoError(t, err)
+	assert.Equal(t, "\"val2\"\n", val)
```

Comment:
Jo tady mám asi poslední dotaz jestli mám stripovat ten `\n` na konci co to vrací.

Reply:
Ked mame test, ze to tam `\n` vzdy je, tak ho mozes nechat.

Reply:
Budem s tym ratat vo worker.